docs/docs/reference/docgen/javascript/auth/types/provider.mdx

```diff
@@ -4,7 +4,7 @@ title: Provider
 sidebar_label: Provider
 description: No description provided.
 displayed_sidebar: referenceSidebar
-custom_edit_url: https://github.com/nhost/nhost/edit/main/packages/core/src/types.ts#L104
+custom_edit_url: https://github.com/nhost/nhost/edit/main/packages/core/dist/types.d.ts#L96
```

Comment:
I'm worried about this part a little bit. Referenced types will be picked up from the `dist` folder after these changes, so line number won't make too much sense here, because it will be totally different from their location in the actual source code. Line numbers will not too easy to determine here.

Reply:
I added the following option to `nhost-js.typedoc.json` to make it work:
```json
{
 // ...
 "compilerOptions": {
    "rootDir": "../..",
    "noImplicitAny": false,
    "paths": {
      "@nhost/core": ["../core/src/index.ts"],
      "@nhost/hasura-auth-js": ["../hasura-auth-js/src/index.ts"],
      "@nhost/hasura-storage-js": ["../hasura-storage-js/index.ts"]
    }
  }
}
```

Reply:
ah ok, I was working on it, please proceed :)
did you notice dependencies `custom_edit_url` paths were rendered with `packages/packages`?

Reply:
Yep but only for docs in auth folder, which is weird.

Reply:
Yes, weird and a bit annoying
Here's an ugly workaround:
```ts
export const editUrl = (sources: Source[] | undefined) => {
  const { baseEditUrl } = snapshot(appState)
  const source = sources?.[0]
  if (baseEditUrl && source) {
    return `${
      source.fileName.startsWith(baseEditUrl.slice(baseEditUrl.lastIndexOf('/') + 1) + '/')
        ? baseEditUrl.slice(0, baseEditUrl.lastIndexOf('/'))
        : baseEditUrl
    }/${source.fileName}L${source.line}`
  }
  return null
}
```

that can be used like this
```
const customEditUrl = editUrl(sources)
`${customEditUrl ? `custom_edit_url: ${customEditUrl}` : ``}`
```

Reply:
Thanks @plmercereau!
https://github.com/nhost/nhost/blob/monorepo-improvements/packages/docgen/src/helpers/mergeUrls.ts

This issue has been resolved.